sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -721,6 +744,11 @@ public partial class BlobLeaseRequestConditions : Azure.RequestConditions
         public BlobLeaseRequestConditions() { }
         public string TagConditions { get { throw null; } set { } }
     }
+    public partial class BlobLegalHoldResult
```

Comment:
does class name match <Operation>Result pattern? see https://azure.github.io/azure-sdk/general_design.html#model-types

Reply:
Let's let One SDK decide in design review.